cmd/jwt/main.go

```diff
@@ -14,14 +14,15 @@ import (
 	"io/ioutil"
 	"os"
 	"regexp"
+	"sort"
 	"strings"
 
 	"github.com/golang-jwt/jwt/v4"
 )
 
 var (
 	// Options
-	flagAlg     = flag.String("alg", "", "signing algorithm identifier")
+	flagAlg     = flag.String("alg", "", fmt.Sprintf("signing algorithm identifier %v", algs()))
```

Comment:
What are your thoughts if we made this a bit easier to read:

```diff
- flagAlg     = flag.String("alg", "", fmt.Sprintf("signing algorithm identifier %v", algs()))
+ flagAlg     = flag.String("alg", "", fmt.Sprintf("signing algorithm identifiers:\n%v", strings.Join(algs(), ",")))
```
Printing out:

```
  -alg string
    	signing algorithm identifiers:
    	ES256,ES384,ES512,EdDSA,HS256,HS384,HS512,PS256,PS384,PS512,RS256,RS384,RS512,none
```

instead of:

```
  -alg string
    	signing algorithm identifiers: ES256,ES384,ES512,EdDSA,HS256,HS384,HS512,PS256,PS384,PS512,RS256,RS384,RS512,none
```

Reply:
Changed to 
```
  -alg string
        signing algorithm identifier, one of
        ES256, ES384, ES512, EdDSA, HS256, HS384, HS512,
        PS256, PS384, PS512, RS256, RS384, RS512, none
```